kube/deploy.sh

```diff
@@ -0,0 +1,42 @@
+export KUBE_NAMESPACE=${KUBE_NAMESPACE}
+export KUBE_SERVER=${KUBE_SERVER}
+
+if [[ -z ${VERSION} ]] ; then
+    export VERSION=${IMAGE_VERSION}
```

Comment:
This will never fire; if `$VERSION` is unset, the script'll quit before the if statement is evaluated.

Reply:
Done.